papaya-web/src/main/java/pl/edu/agh/papaya/mappers/AvailabilityMapper.java

```diff
@@ -2,19 +2,32 @@
 
 import java.time.Duration;
 import java.util.Optional;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 import pl.edu.agh.papaya.api.model.AvailabilityDto;
 import pl.edu.agh.papaya.model.Availability;
+import pl.edu.agh.papaya.service.sprint.SprintService;
 
 @Component
+@RequiredArgsConstructor
 public class AvailabilityMapper implements Mapper<Availability, AvailabilityDto> {
 
+    private final SprintService sprintService;
+
     @Override
     public AvailabilityDto mapToApi(Availability availability) {
+        Long timeAvailable = Optional.ofNullable(availability.getTimeAvailable())
+                .map(Duration::toMinutes)
+                .orElse(0L);
+
+        Double prevAverageSprintCoefficient =
+                sprintService.getPrevSprintAverageCoefficient(availability.getSprint());
+
+        Long effectiveTimeAvailable = (long) (timeAvailable / prevAverageSprintCoefficient);
```

Comment:
To IntelliJ sam mówi, że mogą być prymitywy - w paru innych miejscach też jeszcze